gcsweb/cmd/gcsweb/gcsweb.go

```diff
@@ -487,6 +485,10 @@ func pathPrefix(prowPath *prowv1.ProwPath) string {
 	return joinPath(providerPrefix(prowPath.StorageProvider()), prowPath.Bucket())
 }
 
+func fullObjectPath(prowPath *prowv1.ProwPath) string {
```

Comment:
Let's expose this on `ProwPath` - downstream casts like this will lead to panics if the implementation ever changes.